lib/cdm/cdm/media/cdm/cdm_adapter.cc

```diff
@@ -443,7 +443,13 @@ cdm::Status CdmAdapter::Decrypt(const cdm::InputBuffer_2& encrypted_buffer,
   else if (cdm11_)
     ret = cdm11_->Decrypt(encrypted_buffer, decrypted_buffer);
   else if (cdm10_)
+#ifdef TARGET_WEBOS
+    // we set this to kStreamTypeAudio, as this bypasses automatic SVP header/meta data injection
+    // this is data that is lost when entering the media pipeline
+    ret = cdm10_->Decrypt(encrypted_buffer, decrypted_buffer, cdm::StreamType::kStreamTypeAudio);
```

Comment:
Clearly a hack,
is it worth waiting to merge the PR to see if you make progress towards a more appropriate solution?

and in case IMO it would make sense to apply this also to cdm11 and cdm12 as well,
without having to do another PR in the future to avoid left users in a broken state